test/bucket_test.cc

```diff
@@ -11,9 +11,16 @@
 #include "test_utils.h"
 
 namespace hapi = hermes::api;
+std::shared_ptr<hermes::api::Hermes> hermes_app;
```

Comment:
Why make this a global variable?

Reply:
As the callback defined for compression needs to get the blob.